front/app/containers/Admin/formBuilder/components/FormBuilderSettings/ConfigMultiselectWithLocaleSwitcher/index.tsx

```diff
@@ -109,6 +109,10 @@ const ConfigMultiselectWithLocaleSwitcher = ({
           render={({ field: { ref: _ref, value: choices, onBlur } }) => {
             const canDeleteLastOption =
               allowDeletingAllOptions || choices.length > 1;
+            const validatedValues = choices.map((choice) => ({
+              title_multiloc: choice.title_multiloc,
+            }));
+            debugger;
```

Comment:
@EdwinKato Just a leftover debugger here :)

Reply:
Nice catch, not sure how I missed it 😄